AAI/AAIConnectProfile.md

```diff
@@ -628,6 +635,8 @@ where:
 
 2.  The header MUST NOT contain a `jku`.
 
+3. JWT payload MAY contain `aud` to list approved brokers.
```

Comment:
This section is for headers don't carry `aud` (even though it mentions payload). To solve, delete this line and leave the "payload" line that already exists below.

Let me know if your intent is to replicate claims here: https://tools.ietf.org/html/draft-ietf-oauth-json-web-token-32#section-5.3

This should be done with caution if using the `aud` field.